lms/static/sass/views/_program-list.scss

```diff
@@ -13,6 +13,26 @@
   @include span-columns(12);
 }
 .sidebar{
+    float: right !important;
+    //padding: $baseline;
+    margin-left: 2.35765%;
+    margin-bottom: 20px;
+    width: 23.23176%;
+  .xseries-certificates{
```

Comment:
It would be good if you could move the xseries certificate view styles to a mixin so the view could be easily reused in other contexts (other than just on the sidebar) if needed.